README.md

```diff
@@ -10,6 +10,10 @@ There is also a command line tool called "p6doc".
 (If you are browsing this repository via GitHub, it will not display most
 files correctly, because this is Perl 6 Pod, and GitHub assumes Perl 5 POD).
 
+## Translate README
```

Comment:
Please change this to "README translations" or "README in other languages"

Reply:
OK, I've changed it again!